takhojin/services/userService.js

```diff
@@ -1,14 +1,47 @@
-const bcrypt = require("bcrypt");
-
 const userDao = require("../models/userDao");
 const { pwValidation } = require("../utils/validation-check");
 
+const bcrypt = require("bcrypt");
+
+const jwt = require("jsonwebtoken");
```

Comment:
```suggestion
const bcrypt = require("bcrypt");
const jwt = require("jsonwebtoken");

const userDao = require("../models/userDao");
const { pwValidation } = require("../utils/validation-check");
```
모듈을 import 하는 순서에 대한 컨벤션은 built-in 모듈 - 외부 모듈 - custom 모듈 순서입니다!

Reply:
제가 아직 모듈 구분을 못하는거 같은데
buil-in 모듈은 bycript와 같이 npm에서 install하는 모듈들
외부 모듈은 userDao와 같은 외부에 접근할떄 쓰는 모듈
custom은 제가 만든 모듈들을 의미하나요??